classroom/snippets/src/test/java/TestAcceptInvitation.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestAcceptInvitation {
+
+  private String invitationId = "insert_invitation_id";
+
+  @Test
+  public void testAcceptInvitation() throws GeneralSecurityException, IOException {
```

Comment:
Should there also be a test for calling `AcceptInvitation.acceptInvitation` for an invalid ID?

Reply:
Added.

Reply:
Great, thanks. I think the same applies to the other test files; IMO if the conditions that might throw an exception are controllable and predictable (like a bad `id`), it probably makes sense to include a test that the correct exception is thrown.

Reply:
Added to the invitation test files. Will take an AI to update the other test files in the future.